pkg/platform/platform.go

```diff
@@ -27,15 +27,16 @@ func InfraStatus(client cnoclient.Client) (*bootstrap.InfraStatus, error) {
 	}
 
 	res := &bootstrap.InfraStatus{
-		PlatformType:         infraConfig.Status.PlatformStatus.Type,
-		PlatformStatus:       infraConfig.Status.PlatformStatus,
-		ExternalControlPlane: infraConfig.Status.ControlPlaneTopology == configv1.ExternalTopologyMode,
-		APIServers:           map[string]bootstrap.APIServer{},
+		PlatformType:           infraConfig.Status.PlatformStatus.Type,
+		PlatformStatus:         infraConfig.Status.PlatformStatus,
+		ControlPlaneTopology:   infraConfig.Status.ControlPlaneTopology,
+		InfrastructureTopology: infraConfig.Status.InfrastructureTopology,
+		APIServers:             map[string]bootstrap.APIServer{},
 	}
 
 	// If we use hypershift, the components need proxy settings. In selfhosted, they
 	// are always local to the API so there it is never needed.
-	if res.ExternalControlPlane {
+	if res.ControlPlaneTopology == configv1.ExternalTopologyMode {
```

Comment:
Is this why we're deleting the platform_test.go?

Reply:
See https://github.com/openshift/cluster-network-operator/pull/1524#discussion_r941638270

Reply:
Otherwise, I like the change. Thanks for the patch. OK to move forward if it is indeed correct to be removing `platform_test.go`

Reply:
Thanks, seems OK to me.